### DISCUSSION TITLE
'Used by' section abuse

### DISCUSSION THREAD

**Discussion by PabloLec** [community/community#41535] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hi!

Unlike other sections like `Releases` or `Environments`, we currently cannot hide `Used by`.  
And this quite unfortunate as it seems people may try to use it as a way to gain visibility or whatever.

For instance, my repo https://github.com/PabloLec/RecoverPy hosts a Python project which is an app of its own, not a library or an API of any form which could be used as a dependency.   
But a few weeks ago I started to see some entries in "Used by", 2 or 3, same guy, no possible use of my code in his repos.  
It seems his `requirements.txt` doesn't even contain my project anymore. But from what I saw, he creates repos with huge `requirements` file, probably "popular" Python project to appear in the Used by section.

Maybe I'm an isolated case ( :cry: ) but I guess it would be useful to be able to hide "Used by" section just like the others. This section is not relevant for all repos and as you can see, can be "abused".

**Comment by PabloLec** [comment]
Bump, if anyone has a solution

**Comment by spenserblack** [comment]
Related discussion: https://github.com/orgs/community/discussions/20984

**Reply by PabloLec** [reply]
Thanks :+1: I just reported an abuse to GitHub for this particular user, mentioning the possibility to hide this section, that would be great.
